test/0131/unknown-optional-params.test.js

```diff
@@ -0,0 +1,159 @@
+const { it } = require('@jest/globals');
+const { linterForAepRule } = require('../utils');
+require('../matchers');
+
+let linter;
+
+beforeAll(async () => {
+  linter = await linterForAepRule('0131', 'aep-131-unknown-optional-params');
+  return linter;
+});
+
+test('aep-131-unknown-optional-params should find errors', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
+    paths: {
+      '/test1/{id}': {
+        get: {
+          parameters: [
+            {
+              name: 'force',
+              in: 'query',
+              schema: {
+                type: 'boolean',
+              },
+            },
+          ],
+        },
+      },
+      '/test2/{id}': {
+        parameters: [
+          {
+            name: 'force',
+            in: 'query',
+            schema: {
+              type: 'boolean',
+            },
+          },
+        ],
+        get: {
+          responses: {
+            200: {
+              description: 'OK',
+            },
+          },
+        },
+      },
+    },
+  };
+  return linter.run(oasDoc).then((results) => {
+    expect(results.length).toBe(2);
+    expect(results).toContainMatch({
+      path: ['paths', '/test1/{id}', 'get', 'parameters', '0', 'name'],
+      message: 'A standard Get method should not have unknown optional parameters.',
+    });
+    expect(results).toContainMatch({
+      path: ['paths', '/test2/{id}', 'parameters', '0', 'name'],
+      message: 'A standard Get method should not have unknown optional parameters.',
+    });
+  });
+});
+
+test('aep-131-unknown-optional-params should find no errors', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
+    paths: {
+      // No parameters
+      '/test1/{id}': {
+        patch: {},
+      },
+      // required path parameters, allowed optional parameters
+      // optional header parameters
+      '/test1/{testId}/test2/{id}': {
+        get: {
+          parameters: [
+            {
+              name: 'testId',
```

Comment:
nit: we generally want kebab-case on things like resource names, as well as parameters: https://aep.dev/4/.

doesn't affect the test of course.